### PR TITLE
add `renovate` user test

### PR DESCRIPTION
**Pull request by JamieMagee**
[Renovate][1] is a large TypeScript project with over 11k GitHub stars. Not enough to make it into the top 100 TypeScript repositories on GitHub, although not far off! We frequently see issues with TypeScript updates. Most recently we hit microsoft/TypeScript#50479 with the upgrade to TypeScript 4.8. [Here's a search][2] for TypeScript upgrade PRs sorted by the number of comments.

Given the size and complexity of the Renovate codebase, as well as the frequency of issues we experience when upgrading TypeScript, I believe it is appropriate to include it in the test suite.

[1]: https://github.com/renovatebot/renovate
[2]: https://github.com/renovatebot/renovate/pulls?q=is%3Apr+sort%3Acomments-desc+author%3Aapp%2Frenovate+-is%3Aopen+in%3Atitle+%22update+dependency+typescript%22+

### CODE REVIEW

File: userTests/renovate/test.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "cloneUrl": "https://github.com/renovatebot/renovate.git",
+  "branch": "main",
+  "types": []
+}
```
Comment (HonkingGoose) [inline userTests/renovate/test.json:5]: Looks like there's an end of file error or whitespace error?

Comment (HonkingGoose) [reply userTests/renovate/test.json:5]:
```suggestion
}

```

Looks like we need to add a newline here.

Comment (JamieMagee) [reply userTests/renovate/test.json:5]: Good spot. Thanks!

**Review by jakebailey — APPROVED** [review]

**Comment by jakebailey** [comment]
Thanks!

**Comment by jakebailey** [comment]
Unfortunately I'm going to have to back this out; it doesn't appear to work:

```
Starting #15 / 18: https://github.com/renovatebot/renovate.git
/home/vsts/work/1/s> sudo mount -t tmpfs -o size=4g tmpfs /mnt/ts_downloads
Cloning if absent
Cloning https://github.com/renovatebot/renovate.git into renovate
Installing packages if absent
Ignoring package install error from non-product folder test/e2e:
> Exited with code 1
> No stdout
> error "../../renovate-v0.0.0-semantic-release.tgz": Tarball is not in network and can not be located in cache (["/mnt/ts_downloads/renovate/renovate-v0.0.0-semantic-release.tgz","/home/vsts/.cache/yarn/v6/.tmp/c8dfa9055a0cda93989d5be6b4d41ff9/.yarn-tarball.tgz"])
/mnt/ts_downloads/renovate> yarn cache clean --all
yarn cache v1.22.19
success Cleared cache.
Done in 3.61s.

Testing with /home/vsts/work/1/s/typescript-53146/built/local/tsserver.js (new)
Killing process 3748 and its descendents: 3759
New server timed out after 600000 ms
Done https://github.com/renovatebot/renovate.git
Repo https://github.com/renovatebot/renovate.git had status "Unknown failure"
Cleaning up repo
/home/vsts/work/1/s> sudo umount /mnt/ts_downloads
```

I haven't looked into why, but, it's blocking us from seeing green changes.

**Comment by JamieMagee** [comment]
Yep, makes sense. It appears to be our E2E tests that are causing the failure. I'll investigate and open another PR when I can get it working.

**Pull request merged**